src/hazelweb/Update.re

```diff
@@ -117,7 +117,7 @@ let apply_action =
           JSUtil.log("[Program.MissingCursorInfo]");
           model;
         | exception Program.InvalidInput =>
-          JSUtil.log("[Program.InvalidInput");
```

Comment:
Should we be showing errors for any of these?  Maybe not a full crash message but something so developers bother to report or track these down when they occur.

- Program.FailedAction
- Program.CursorEscaped
- Program.MissingCursorInfo
- Program.InvalidInput
- Program.DoesNotElaborate

Reply:
Yes, ideally all of the invariant violation exceptions get reported this way so that we get reports more specific than just "Hazel froze/crashed"